src/components/AddBook.jsx

```diff
@@ -2,15 +2,19 @@ import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { nanoid } from 'nanoid';
 import { addBook, getBooks } from '../redux/books/booksSlice';
+import '../styles/AddBook.css';
 
 const AddBook = () => {
   //  const { books } = useSelector((store) => store.books);
```

Comment:
- [ ] Please kindly remove all commented code from your codebase as it is not a good practice.